examples/ubuntu-nlb-consul-primary/variables.tf

```
@@ -12,7 +12,6 @@ variable "project_id" {
 variable "region" {
   type        = string
   description = "(optional) The region to host the cluster in"
```

Looks like this is required now? We could also omit all of the `region` parameters in the module code and rely on the provider configuration (aka the way all AWS code used to be).
```suggestion
  description = "(required) The region to host the cluster in"
```